feature/pom.xml

```diff
@@ -110,7 +110,6 @@ limitations under the License.
                 <feature>jclouds-compute</feature>
                 <feature>jclouds-api-atmos</feature>
                 <feature>jclouds-api-byon</feature>
-                <feature>jclouds-api-cloudsigma2</feature>
```

Comment:
I've seen you already defined the `jclouds-all-blobstore` and `jclouds-all-compute`. Worth just adding the super-features here now?